### PR TITLE
add skipped indication to pylint

### PR DESCRIPTION
**Pull request by glicht**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/25209

## Description
Add skipped indication to pylint

## Screenshots
Paste here any images that will help the reviewer

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -22,6 +22,7 @@
 * Added validation to ensure that integrations, scripts, and playbooks do not contain the entity type in their names.
 * Added the **convert** command to convert entities between XSOAR versions.
 * Added the *--deprecate* flag in **format** command to deprecate integrations, scripts, and playbooks.
+* Added to `pylint` summary an indication if a test was skipped.
```
Comment (roysagi) [inline CHANGELOG.md:25]: this changelog entry is in the incorrect location please move it.

**Review by roysagi — APPROVED** [review]
Please change the changelog location to the correct header.

**Review by yaakovi — APPROVED** [review]
please add a screenshot of the change in the build

**Comment by glicht** [comment]
> please add a screenshot of the change in the build

@yaakovi see here: https://github.com/demisto/etc/issues/25209#issuecomment-864577974

**Pull request merged**